source/_integrations/aws_s3.markdown

```diff
@@ -24,10 +24,64 @@ Secret Access Key:
   description: "The secret access key for your AWS S3 account."
 Bucket Name:
   description: "S3 bucket name to store the backups. Bucket must already exist and be writable by the provided credentials."
+Prefix:
+  description: Prefix to use for the backup keys in the bucket. This is optional.⚠️ Be sure to use a unique, non-overlapping prefix to avoid issues with backup assignment. See [known limitation](#assignment-of-backups-to-home-assistant-instances-and-config-entries).
```

Comment:
```suggestion
  description: Prefix to use for the backup keys in the bucket. This is optional. Be sure to use a unique, non-overlapping prefix to avoid issues with backup assignment. See [known limitation](#assignment-of-backups-to-home-assistant-instances-and-config-entries).
```
Please remove the icons. We don't usually use icons in the documentation unless they refer to [icons used in the UI](https://developers.home-assistant.io/docs/documenting/integration-docs-examples#inline-icons) or are part of the documentation template.